components/cardano-node/dowload-mithril-snapshot.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+COMPONENT_DIR="$SCRIPT_DIR/.."
+ROOT_DIR="$COMPONENT_DIR/.."
+
+# shellcheck source=/dev/null
+source "${ROOT_DIR}/configurations/mithril-configs/${NETWORK}.env"
+
+rm -rf "$ROOT_DIR/db"
+
+mithril-client snapshot download latest
+
+mv "$ROOT_DIR/db/*" "$ROOT_DIR/mithril-snapshot"
```

Comment:
```suggestion
mv "$ROOT_DIR/db" "$ROOT_DIR/mithril-snapshot"
```

Did you mean this?

Reply:
The one you suggested creates a `db` folder inside `mithril-snapshot`, I only want to move the contents of the db folder